src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/Type.cs

```diff
@@ -404,7 +366,7 @@ public ConstValKind constValKind()
         }
         public CType underlyingType()
         {
-            if (IsAggregateType() && getAggregate().IsEnum())
+            if (this is AggregateType && getAggregate().IsEnum())
```

Comment:
Do you have a bug tracking this somewhere?

---
In reply to: [128341959](https://github.com/dotnet/corefx/pull/22446#discussion_r128341959) [](ancestors = 128341959)